packages/react-components/react-card/src/components/Card/useCardStyles.ts

```diff
@@ -24,9 +25,14 @@ export const cardCSSVars = {
 
 const useStyles = makeStyles({
   root: {
+    ...shorthands.overflow('hidden'),
+    ...shorthands.borderRadius(`var(${cardCSSVars.cardBorderRadiusVar})`),
+    ...shorthands.padding(`var(${cardCSSVars.cardSizeVar})`),
+    ...shorthands.gap(`var(${cardCSSVars.cardSizeVar})`),
+
     display: 'flex',
     position: 'relative',
-    ...shorthands.overflow('hidden'),
+    boxSizing: 'border-box',
```

Comment:
This is to prevent the card to get wrong sizes when width/height styles are applied to the Card component directly (in the userland).
The Card paddings can be 8px, 12px or 16px. If, in the userland, the card receives a `width: 300px`, then the actual width will be 300px + padding * 2, which will break the box model.